<ui version="4.0" >
 <author>Harm van Eersel &lt;devsciurus@xs4all.nl></author>
 <class>SettingsDialog</class>
 <widget class="QDialog" name="SettingsDialog" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>548</width>
    <height>459</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>molsKetch - Preferences</string>
  </property>
  <property name="windowIcon" >
   <iconset resource="molsketch.qrc" >:/images/configure.png</iconset>
  </property>
  <widget class="QWidget" name="layoutWidget" >
   <property name="geometry" >
    <rect>
     <x>9</x>
     <y>9</y>
     <width>531</width>
     <height>441</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" >
    <item>
     <layout class="QHBoxLayout" >
      <item>
       <widget class="QListWidget" name="listWidget" >
        <property name="sizePolicy" >
         <sizepolicy vsizetype="Expanding" hsizetype="Fixed" >
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="maximumSize" >
         <size>
          <width>88</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="iconSize" >
         <size>
          <width>32</width>
          <height>32</height>
         </size>
        </property>
        <property name="movement" >
         <enum>QListView::Static</enum>
        </property>
        <property name="spacing" >
         <number>12</number>
        </property>
        <property name="viewMode" >
         <enum>QListView::IconMode</enum>
        </property>
        <property name="wordWrap" >
         <bool>true</bool>
        </property>
        <property name="currentRow" >
         <number>-1</number>
        </property>
        <item>
         <property name="text" >
          <string>General</string>
         </property>
         <property name="icon" >
          <iconset resource="molsketch.qrc" >:/images/configure.svg</iconset>
         </property>
        </item>
        <item>
         <property name="text" >
          <string>Draw settings</string>
         </property>
         <property name="icon" >
          <iconset resource="molsketch.qrc" >:/images/draw-freehand.svg</iconset>
         </property>
        </item>
        <item>
         <property name="text" >
          <string>Libraries</string>
         </property>
         <property name="icon" >
          <iconset resource="molsketch.qrc" >:/images/document-open.svg</iconset>
         </property>
        </item>
       </widget>
      </item>
      <item>
       <widget class="QStackedWidget" name="stackedWidget" >
        <property name="currentIndex" >
         <number>0</number>
        </property>
        <widget class="QWidget" name="generalPage" >
         <widget class="QGroupBox" name="groupBoxGeneral" >
          <property name="geometry" >
           <rect>
            <x>10</x>
            <y>10</y>
            <width>411</width>
            <height>391</height>
           </rect>
          </property>
          <property name="title" >
           <string>Save options</string>
          </property>
          <widget class="QWidget" name="layoutWidget" >
           <property name="geometry" >
            <rect>
             <x>20</x>
             <y>40</y>
             <width>376</width>
             <height>105</height>
            </rect>
           </property>
           <layout class="QVBoxLayout" >
            <item>
             <layout class="QHBoxLayout" >
              <item>
               <widget class="QLabel" name="labelDefaultFileType" >
                <property name="text" >
                 <string>Default file type</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QComboBox" name="comboBoxFileType" >
                <item>
                 <property name="text" >
                  <string>MDL (*.mdl)</string>
                 </property>
                </item>
                <item>
                 <property name="text" >
                  <string>Chemical Markup Language (*.cml)</string>
                 </property>
                </item>
               </widget>
              </item>
             </layout>
            </item>
            <item>
             <layout class="QHBoxLayout" >
              <item>
               <widget class="QLabel" name="labelDefaultImageType" >
                <property name="text" >
                 <string>Default image type</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QComboBox" name="comboBoxImageType" >
                <item>
                 <property name="text" >
                  <string>Scalable Vector Graphics (*.svg)</string>
                 </property>
                </item>
                <item>
                 <property name="text" >
                  <string>Portable Network Graphics (*.png)</string>
                 </property>
                </item>
               </widget>
              </item>
             </layout>
            </item>
            <item>
             <layout class="QHBoxLayout" >
              <item>
               <widget class="QLabel" name="labelAutoSave" >
                <property name="text" >
                 <string>Autosave each</string>
                </property>
               </widget>
              </item>
              <item>
               <spacer>
                <property name="orientation" >
                 <enum>Qt::Horizontal</enum>
                </property>
                <property name="sizeHint" >
                 <size>
                  <width>192</width>
                  <height>25</height>
                 </size>
                </property>
               </spacer>
              </item>
              <item>
               <widget class="QSpinBox" name="spinBoxAutoSave" >
                <property name="suffix" >
                 <string> min</string>
                </property>
                <property name="prefix" >
                 <string/>
                </property>
               </widget>
              </item>
             </layout>
            </item>
           </layout>
          </widget>
         </widget>
        </widget>
        <widget class="QWidget" name="drawingPage" >
         <widget class="QWidget" name="layoutWidget" >
          <property name="geometry" >
           <rect>
            <x>10</x>
            <y>10</y>
            <width>430</width>
            <height>381</height>
           </rect>
          </property>
          <layout class="QVBoxLayout" >
           <item>
            <widget class="QGroupBox" name="groupBoxAtom" >
             <property name="sizePolicy" >
              <sizepolicy vsizetype="Preferred" hsizetype="Expanding" >
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="title" >
              <string>Atom settings</string>
             </property>
             <property name="checkable" >
              <bool>false</bool>
             </property>
             <layout class="QGridLayout" >
              <property name="leftMargin" >
               <number>9</number>
              </property>
              <property name="topMargin" >
               <number>9</number>
              </property>
              <property name="rightMargin" >
               <number>9</number>
              </property>
              <property name="bottomMargin" >
               <number>9</number>
              </property>
              <property name="horizontalSpacing" >
               <number>6</number>
              </property>
              <property name="verticalSpacing" >
               <number>6</number>
              </property>
              <item row="2" column="0" >
               <widget class="QCheckBox" name="checkBoxShowHydrogen" >
                <property name="text" >
                 <string>Show neutral hydrogen atoms</string>
                </property>
               </widget>
              </item>
              <item row="0" column="0" >
               <widget class="QCheckBox" name="checkBoxAutoHydrogen" >
                <property name="text" >
                 <string>Automaticly add hydrogens</string>
                </property>
               </widget>
              </item>
              <item row="3" column="0" >
               <widget class="QCheckBox" name="checkBoxShowValency" >
                <property name="sizePolicy" >
                 <sizepolicy vsizetype="Fixed" hsizetype="Preferred" >
                  <horstretch>0</horstretch>
                  <verstretch>0</verstretch>
                 </sizepolicy>
                </property>
                <property name="text" >
                 <string>Show atomic charge</string>
                </property>
               </widget>
              </item>
              <item row="1" column="0" >
               <widget class="QCheckBox" name="checkBoxShowCarbon" >
                <property name="text" >
                 <string>Show neutral carbon atoms</string>
                </property>
               </widget>
              </item>
              <item row="6" column="0" >
               <layout class="QHBoxLayout" >
                <item>
                 <widget class="QDoubleSpinBox" name="doubleSpinBoxFontSize" >
                  <property name="buttonSymbols" >
                   <enum>QAbstractSpinBox::UpDownArrows</enum>
                  </property>
                  <property name="accelerated" >
                   <bool>true</bool>
                  </property>
                  <property name="correctionMode" >
                   <enum>QAbstractSpinBox::CorrectToNearestValue</enum>
                  </property>
                  <property name="suffix" >
                   <string> pt</string>
                  </property>
                  <property name="decimals" >
                   <number>0</number>
                  </property>
                  <property name="minimum" >
                   <double>8.000000000000000</double>
                  </property>
                  <property name="maximum" >
                   <double>120.000000000000000</double>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QFontComboBox" name="fontComboBox" />
                </item>
                <item>
                 <widget class="QPushButton" name="pushButtonFont" >
                  <property name="text" >
                   <string>Details...</string>
                  </property>
                 </widget>
                </item>
               </layout>
              </item>
              <item row="5" column="0" >
               <widget class="QLabel" name="labelAtomSymbolFont" >
                <property name="text" >
                 <string>AtomSymbol font</string>
                </property>
               </widget>
              </item>
              <item row="4" column="0" >
               <spacer>
                <property name="orientation" >
                 <enum>Qt::Vertical</enum>
                </property>
                <property name="sizeType" >
                 <enum>QSizePolicy::Fixed</enum>
                </property>
                <property name="sizeHint" >
                 <size>
                  <width>20</width>
                  <height>20</height>
                 </size>
                </property>
               </spacer>
              </item>
             </layout>
            </widget>
           </item>
           <item>
            <widget class="QGroupBox" name="groupBoxBond" >
             <property name="sizePolicy" >
              <sizepolicy vsizetype="Preferred" hsizetype="Preferred" >
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="title" >
              <string>MsKBond settings</string>
             </property>
             <widget class="QLabel" name="labelBondLength" >
              <property name="geometry" >
               <rect>
                <x>11</x>
                <y>30</y>
                <width>211</width>
                <height>20</height>
               </rect>
              </property>
              <property name="text" >
               <string>MsKBond length:</string>
              </property>
             </widget>
             <widget class="QLineEdit" name="lineEditBondLength" >
              <property name="geometry" >
               <rect>
                <x>230</x>
                <y>30</y>
                <width>171</width>
                <height>26</height>
               </rect>
              </property>
             </widget>
             <widget class="QLabel" name="labelBondWidth" >
              <property name="geometry" >
               <rect>
                <x>11</x>
                <y>61</y>
                <width>221</width>
                <height>19</height>
               </rect>
              </property>
              <property name="text" >
               <string>MsKBond width:</string>
              </property>
             </widget>
             <widget class="QDoubleSpinBox" name="doubleSpinBoxBondWidth" >
              <property name="geometry" >
               <rect>
                <x>230</x>
                <y>60</y>
                <width>171</width>
                <height>25</height>
               </rect>
              </property>
              <property name="suffix" >
               <string> px</string>
              </property>
              <property name="decimals" >
               <number>0</number>
              </property>
              <property name="minimum" >
               <double>1.000000000000000</double>
              </property>
              <property name="maximum" >
               <double>100.000000000000000</double>
              </property>
              <property name="singleStep" >
               <double>1.000000000000000</double>
              </property>
              <property name="value" >
               <double>1.000000000000000</double>
              </property>
             </widget>
             <widget class="QDoubleSpinBox" name="doubleSpinBoxBondAngle" >
              <property name="geometry" >
               <rect>
                <x>230</x>
                <y>90</y>
                <width>171</width>
                <height>25</height>
               </rect>
              </property>
              <property name="suffix" >
               <string>°</string>
              </property>
              <property name="decimals" >
               <number>0</number>
              </property>
              <property name="minimum" >
               <double>6.000000000000000</double>
              </property>
              <property name="maximum" >
               <double>180.000000000000000</double>
              </property>
              <property name="singleStep" >
               <double>6.000000000000000</double>
              </property>
              <property name="value" >
               <double>30.000000000000000</double>
              </property>
             </widget>
             <widget class="QLabel" name="labelBondAngle" >
              <property name="geometry" >
               <rect>
                <x>11</x>
                <y>91</y>
                <width>221</width>
                <height>19</height>
               </rect>
              </property>
              <property name="text" >
               <string>MsKBond angle:</string>
              </property>
             </widget>
            </widget>
           </item>
          </layout>
         </widget>
        </widget>
        <widget class="QWidget" name="librariesPage" >
         <widget class="QGroupBox" name="groupBoxLibraries" >
          <property name="geometry" >
           <rect>
            <x>10</x>
            <y>10</y>
            <width>411</width>
            <height>381</height>
           </rect>
          </property>
          <property name="title" >
           <string>Libraries</string>
          </property>
          <widget class="QWidget" name="layoutWidget" >
           <property name="geometry" >
            <rect>
             <x>20</x>
             <y>40</y>
             <width>371</width>
             <height>131</height>
            </rect>
           </property>
           <layout class="QVBoxLayout" >
            <item>
             <layout class="QHBoxLayout" >
              <item>
               <widget class="QLabel" name="labelLibraryPath" >
                <property name="text" >
                 <string>Library path</string>
                </property>
               </widget>
              </item>
              <item>
               <spacer>
                <property name="orientation" >
                 <enum>Qt::Horizontal</enum>
                </property>
                <property name="sizeHint" >
                 <size>
                  <width>40</width>
                  <height>20</height>
                 </size>
                </property>
               </spacer>
              </item>
              <item>
               <widget class="QLineEdit" name="lineEditLibraryPath" />
              </item>
              <item>
               <widget class="QToolButton" name="toolButtonBrowseLibraryPath" >
                <property name="text" >
                 <string>...</string>
                </property>
               </widget>
              </item>
             </layout>
            </item>
            <item>
             <layout class="QHBoxLayout" >
              <item>
               <widget class="QLabel" name="labelCustomLibraryPath" >
                <property name="text" >
                 <string>Custom library Path</string>
                </property>
               </widget>
              </item>
              <item>
               <spacer>
                <property name="orientation" >
                 <enum>Qt::Horizontal</enum>
                </property>
                <property name="sizeHint" >
                 <size>
                  <width>40</width>
                  <height>20</height>
                 </size>
                </property>
               </spacer>
              </item>
              <item>
               <widget class="QLineEdit" name="lineEditCustomLibraryPath" />
              </item>
              <item>
               <widget class="QToolButton" name="toolButtonBrowseCustomLibraryPath" >
                <property name="text" >
                 <string>...</string>
                </property>
               </widget>
              </item>
             </layout>
            </item>
           </layout>
          </widget>
         </widget>
        </widget>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QDialogButtonBox" name="buttonBox" >
      <property name="orientation" >
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="standardButtons" >
       <set>QDialogButtonBox::Apply|QDialogButtonBox::Cancel|QDialogButtonBox::Help|QDialogButtonBox::NoButton|QDialogButtonBox::Ok|QDialogButtonBox::RestoreDefaults</set>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <tabstops>
  <tabstop>buttonBox</tabstop>
 </tabstops>
 <resources>
  <include location="molsketch.qrc" />
 </resources>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>SettingsDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>366</x>
     <y>431</y>
    </hint>
    <hint type="destinationlabel" >
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>SettingsDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>434</x>
     <y>431</y>
    </hint>
    <hint type="destinationlabel" >
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>listWidget</sender>
   <signal>currentRowChanged(int)</signal>
   <receiver>stackedWidget</receiver>
   <slot>setCurrentIndex(int)</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>62</x>
     <y>21</y>
    </hint>
    <hint type="destinationlabel" >
     <x>260</x>
     <y>12</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
